pkg/skaffold/docker/parse_test.go

```diff
@@ -159,7 +159,7 @@ func TestGetDockerfileDependencies(t *testing.T) {
 			description: "add dependency",
 			dockerfile:  addDockerfile,
 			workspace:   "docker",
-			expected:    []string{"docker/Dockerfile", "docker/nginx.conf"},
+			expected:    []string{"Dockerfile", "nginx.conf"},
```

Comment:
This is worry some for me. I don't have a lot of context, but earlier GetDockerfileDependencies used to return absolute paths.
Now you dont.  Would this not break the workflow.

Reply:
It shouldn't but yes, that's a big change. One that's needed

Reply:
We need to keep things as relative as possible. Each time we switch to absolute paths, it's a mess

Reply:
yes i agree absolute paths are a mess.
Will take one more look.